numba/tests/test_parfors.py

```diff
@@ -2545,6 +2547,16 @@ def test_impl(X):
 
         self.check(test_impl, np.ones(10))
 
+    @skip_unsupported
+    def test_parfor_slice20(self):
+        # issues #4075, slice size
+        def test_impl():
```

Comment:
I'm wondering whether it would be possible to add some more tests for the proposed change into `numba.tests.test_array_analysis`, perhaps some tests that are solely related to the equivalence analysis changes?

Reply:
Done.